examples/example_linearity.ipynb

```diff
@@ -0,0 +1,217 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f68fb2e8-a025-4a7f-a6b3-ad415b5e55b9",
+   "metadata": {},
+   "source": [
+    "# Example: Varying Injection Volume (Linearity)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26e92c78-694a-4c84-82dc-fb1cb27efd65",
+   "metadata": {},
+   "source": [
+    "Here outlines a basic linearity experiment where one sample (and therefore one vial location) is injected at varying injection volumes. \n",
+    "* Varies injection volume within a range in a defined incerement\n",
```

Comment:
```suggestion
    "* Varies injection volume within a range in a defined increment\n",
```

Reply:
Is this fixed? It doesn't seem that way in my viewer, and I think I have the newest version.

There's also a typo in the line below.